### PR TITLE
Official 1.30 Release Docs

### PR DESCRIPTION
**Pull request by drewhagen**
## 🛑 DO NOT MERGE 🛑 UNTIL KUBERNETES 1.30 IS RELEASED
Target Date: 2024-04-17

### Release deadlines
From [sig-release](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.30#timeline):

| Description	| Date |
| ------------- | ---- |
| Docs deadline — Open placeholder PRs |	Thursday 22nd February 2024 |
| Docs deadline — PRs ready for review   |	Tuesday 12th March 2024       |
| Docs Freeze — All PRs reviewed and ready to merge	| Tuesday 26th March 2024 |
| **v1.29.0 released**	| Wednesday 17th April 2024 |

### For More Information, please reach out to the SIG Release Docs team:
cc: @drewhagen

/hold

### CODE REVIEW

**Comment by drewhagen** [comment]
Hey y'all @kubernetes/sig-docs-en-owners @kubernetes/sig-docs-blog-owners
This is the PR to merge the integration branch into main for the Kubernetes website tomorrow, after 1.30 has been realized. Can you look this over to make sure nothing looks out of order?

Thanks!

**Comment by sftim** [comment]
Whoops, we missed asking for reviews on this PR.

**Comment by sftim** [comment]
I see that https://deploy-preview-44710--kubernetes-io-main-staging.netlify.app/releases/download/#binaries is blank; we should make sure that this doesn't stay blank post release.

File: static/_redirects [hunk 1]
```diff
@@ -79,6 +79,7 @@
 /docs/concepts/jobs/run-to-completion-finite-workloads/      /docs/concepts/workloads/controllers/job/ 301
 /id/docs/concepts/jobs/run-to-completion-finite-workloads/      /id/docs/concepts/workloads/controllers/job/ 301
 /docs/concepts/nodes/node/     /docs/concepts/architecture/nodes/ 301
+/docs/storage-force-detach-on-timeout/ /docs/concepts/architecture/nodes/#storage-force-detach-on-timeout 302
```
Comment (sftim) [inline static/_redirects:82]: I wonder where this has come from

**Comment by sftim** [comment]
This looks like a valid change, but: I don't see the updates for generated documentation. Are we happy we'll get those in?

**Comment by drewhagen** [comment]
> Whoops, we missed asking for reviews on this PR.

I will modify the SIG Release Docs handbook so that we review this more in advance in the next cycle.
I'm following guidance under "Day before Release":

https://github.com/kubernetes/sig-release/blob/b51237ddb556bb6d24e43d7424e55cb9e66694d5/release-team/role-handbooks/docs/Release-Timeline.md#get-approvals-for-open-prs

**Comment by drewhagen** [comment]
Oh wait, my apologies, as it looks like I pinged the wrong groups.
@kubernetes/sig-docs-en-reviews Can you give this a review before we merge this in a few hours?

**Comment by drewhagen** [comment]
> This looks like a valid change, but: I don't see the updates for generated documentation. Are we happy we'll get those in?

x-post of our plan: https://kubernetes.slack.com/archives/CA1MMM1HU/p1712071114613859

@tengqm have you generated the docs?

File: static/_redirects (hunk 1 above)
Comment (dipesh-rawat) [reply static/_redirects:82]: Looks like it was added as part of PR https://github.com/kubernetes/website/pull/45119

**Pull request merged**